x-pack/plugins/lens/public/visualizations/metric/toolbar.tsx

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback } from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiFlexGroup, EuiFormRow, EuiFieldText } from '@elastic/eui';
+import { VisualizationToolbarProps } from '../../types';
+import { ToolbarPopover, useDebouncedValue } from '../../shared_components';
+import { MetricVisualizationState } from './visualization';
+
+export function Toolbar(props: VisualizationToolbarProps<MetricVisualizationState>) {
+  const { state, setState } = props;
+
+  const setSubtitle = useCallback(
+    (prefix: string) => setState({ ...state, subtitle: prefix }),
+    [setState, state]
+  );
+
+  const { inputValue: subtitleInputVal, handleInputChange: handleSubtitleChange } =
+    useDebouncedValue<string>(
+      {
+        onChange: setSubtitle,
+        value: state.subtitle || '',
+      },
+      { allowFalsyValue: true }
+    );
+
+  const hasBreakdownBy = Boolean(state.breakdownByAccessor);
+
+  return (
+    <EuiFlexGroup alignItems="center" gutterSize="none" responsive={false}>
+      {!hasBreakdownBy && (
+        <ToolbarPopover
+          title={i18n.translate('xpack.lens.metric.labels', {
+            defaultMessage: 'Labels',
+          })}
+          type="labels"
+          groupPosition="left"
```

Comment:
```suggestion
          groupPosition="none"
```
to change this to this:
<img width="61" alt="Screenshot 2022-07-26 at 14 55 58" src="https://user-images.githubusercontent.com/4283304/181011008-d905a57d-4d59-4184-925b-6dd4cd579c3d.png">
<img width="58" alt="Screenshot 2022-07-26 at 14 55 42" src="https://user-images.githubusercontent.com/4283304/181011034-cc1673e3-bcf3-4560-abb5-53b426a377ad.png">